src/routes/business.js

```diff
@@ -99,7 +99,7 @@ router.delete('/:id', async (request, response, next) => {
     next();
   }
 });
-
+//find user's favorite businesses. this isn't working for me, i got a prisma error
 router.get('/favorites', async (request, response, next) => {
```

Comment:
Thank you for letting me know, but how did you test this? Did you use Postman? What Prisma error pops up?